testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -617,4 +617,32 @@ public void testSetViewAuthorizationWithSecurityInvoker()
     {
         assertQuerySucceeds("ALTER VIEW mock.default.test_view_invoker SET AUTHORIZATION some_other_user");
     }
+
+    @Test
+    public void testUseStatementAccessControl()
```

Comment:
nit: We can divide this test method into few smaller ones.

Reply:
Spitted into 3 test methods